### PR TITLE
Update secure-your-business-data.md

### PR DESCRIPTION
**Pull request by msbemba**
Setting the "Notify" action for the transport rule "Anti-ransomware rule: warn users" results in double-up emails that confuse recipients and do not provide a clear message about the action required. Instead changed it to '"Prepend a Disclaimer" so that the warning is stamped onto the original email message, making it clear what content the warning applies to, and providing a more familiar experience to the user.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2643

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/security-and-compliance/secure-your-business-data.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ To create a mail transport rule, view a [short training video](https://support.m
 |Name  <br/> |Anti-ransomware rule: warn users  <br/> |Anti-ransomware rule: block file types  <br/> |
 |Apply this rule if . . .  <br/> |Any attachment . . . file extension matches . . .  <br/> |Any attachment . . . file extension matches . . .  <br/> |
 |Specify words or phrases  <br/> |Add these file types:  <br/> dotm, docm, xlsm, sltm, xla, xlam, xll, pptm, potm, ppam, ppsm, sldm  <br/> |Add these file types:  <br/> ade, adp, ani, bas, bat, chm, cmd, com, cpl, crt, hlp, ht, hta, inf, ins, isp, job, js, jse, lnk, mda, mdb, mde, mdz, msc, msi, msp, mst, pcd, reg, scr, sct, shs, url, vb, vbe, vbs, wsc, wsf, wsh, exe, pif  <br/> |
-|Do the following . . .  <br/> |Notify the recipient with a message  <br/> |Block the message . . . reject the message and include an explanation  <br/> |
+|Do the following . . .  <br/> |Prepend a Disclaimer  <br/> |Block the message . . . reject the message and include an explanation  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/secure-your-business-data.md]:
```suggestion
|Do the following . . .  <br/> |Prepend a disclaimer  <br/> |Block the message . . . reject the message and include an explanation  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**